packages/web-components/fast-foundation/src/tabs/tabs.spec.ts

```diff
@@ -293,14 +293,49 @@ describe("Tabs", () => {
 
             await connect();
 
-            element.setAttribute("activeId", "03")
+            element.setAttribute("activeId", "03");
```

Comment:
Sounds like `prettier` added this, guess we should keep it

Reply:
yea likely